jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/TravelerMessageFromHumanToAsnConverter.java

```diff
@@ -1,33 +1,33 @@
 /*******************************************************************************
- * Copyright 2018 572682
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * Copyright 2018 572682.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not

```

Comment:
Why the html addition?

Reply:
This was one of the checkstyle warnings, likely because the rule is part of the Google style guide we used as a foundation.